src/assets/AssetsDetectionController.ts

```diff
@@ -150,15 +150,11 @@ export class AssetsDetectionController extends BaseController<
     try {
       const openSeaApiKey = this.getOpenSeaApiKey();
       /* istanbul ignore if */
-      if (openSeaApiKey) {
-        response = await timeoutFetch(
-          api,
-          { headers: { 'X-API-KEY': openSeaApiKey } },
-          15000,
-        );
-      } else {
-        response = await timeoutFetch(api, {}, 15000);
-      }
+      response = await timeoutFetch(
+        api,
+        openSeaApiKey ? { headers: { 'X-API-KEY': openSeaApiKey } } : {},
+        15000,
+      );
```

Comment:
@sethkfman Does the API no longer allow requests without an API key? Was that else block never really used?

Reply:
@adonesky1 It looks like it is still optional. Per the OpenSea Documentation: 
```
X-API-KEY (string)
Your optional API Key (contact us to request one!)
```